roles/user-management/manage-idm-users/tasks/configure_user.yml

```diff
@@ -12,6 +12,7 @@
 # The addition of expiration date is a request that will be submitted upstream
 #      krbprincipalexpiration: "{{ item.expiration_date | default('') }}"
     with_items: "{{ users }}"
+      state: "{{ item.state | default('present') }}"
```

Comment:
I believe this `state` applies to the `ipa_user` above, so please move it above the `with_items` line.